sdk/src/main/java/co/omise/android/models/PaymentMethod.kt

```diff
@@ -0,0 +1,43 @@
+package co.omise.android.models
+
+import android.os.Parcel
+import android.os.Parcelable
+import org.json.JSONObject
+
+class PaymentMethod : Model {
+    var name: String? = null
+    var currencies: List<String>? = null
+    var cardBrands: List<String>? = null
+    var installmentTerms: List<Int>? = null
+
+
+    constructor(rawJson: String) : this(JSONObject(rawJson))
+
+    private constructor(json: JSONObject) : super(json) {
+        name = JSON.string(json, "name")
+        currencies = JSON.stringList(json, "currencies")
+        cardBrands = JSON.stringList(json, "card_brands")
+        installmentTerms = JSON.integerList(json, "installment_terms")
+    }
+
+    override fun writeToParcel(parcel: Parcel, flags: Int) {
+        super.writeToParcel(parcel, flags)
+        parcel.writeString(name)
+        parcel.writeStringList(currencies)
+        parcel.writeStringList(cardBrands)
+    }
+
+    override fun describeContents(): Int {
+        return 0
```

Comment:
What is this property for?

Reply:
Just carries on from parent `Model` class that has `Parcelable` implemented in it.

Reply:
This fix 0 value looks like a code smell to me. Do you know what is this property used for? So that we can know how to properly implement it.